### DISCUSSION TITLE
Using environment variable to configure client does not work for me

### DISCUSSION THREAD

**Discussion by C1oudNinja** [matehat/cqerl#171] [Q&A]
Hello I've a short question. I wanted to set cassandra_nodes as environment variable. But it does not work. 
Can you provide an example how the environment variable should look. 
I tested it with:
`cassandra_nodes=[{"127.0.0.1", 9042 }`

But then i received: 

neighbours crashed with reason: no match of right hand value {error,cluster_not_configured}

Thank you in advance.

**Comment by matehat** [comment]
If you mean that you are going to set an environment variable that you'd set either:

- in Docker, as:

    ```docker
    ENV CASSANDRA_NODES="127.0.0.1:9042,127.0.0.1:9043"
    ```

- in a shell script, as:

    ```shell
    export CASSANDRA_NODES="127.0.0.1:9042,127.0.0.1:9043"
    ```

etc, then you'd need to obtain those and set those when starting your application, like this:

```erlang
start(_StartType, _StartArgs) ->
    NodesVar = os:getenv("CASSANDRA_NODES"),
    Nodes = string:tokens(NodesVar, ","),
    application:ensure_all_started(cqerl),
    cqerl_cluster:add_nodes(Nodes).
```
